src/main/java/org/mule/extension/db/api/param/JdbcType.java

```diff
@@ -97,6 +97,8 @@
 
   SQLXML(new ResolvedDbType(Types.SQLXML, "SQLXML")),
 
+  REF_CURSOR(new ResolvedDbType(Types.SQLXML, "REF_CURSOR")),
```

Comment:
I think it should be, 
`REF_CURSOR(new ResolvedDbType(Types.REF_CURSOR, "REF_CURSOR")),`